pkg/cmd/create_cluster.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023 The Kubebb Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCreateClusterCmd() *cobra.Command {
```

Comment:
For the 1st version,should we manage cluster ?

Reply:
If we need cli to manage clusters, I think we need to add handling of clusters and can just introduce kind as our subcommand.